src/modules/Home/hooks/useAddProjectForm.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { ProjectFormData } from '../types';
+
+export function useAddProjectForm(): [
+  ProjectFormData,
+  React.Dispatch<ProjectFormData>,
+  boolean,
+  React.Dispatch<boolean>,
+] {
+  const initialForm: ProjectFormData = {
```

Comment:
Move this constant outside the function otherwise it's created on every call of this function